Src/AutoFixtureUnitTest/Kernel/FiniteSequenceRequestTest.cs

```diff
@@ -19,20 +19,19 @@ public void InitializeWithNullRequestAndValidNumberThrows()
             // Teardown
         }
 
-        [Theory]
-        [InlineData(0)]
-        [InlineData(-1)]

```

Comment:
Instead of removing these test cases entirely, would it be possible to just replace the one with the 0 with another negative number? For example -27, similar to what @ploeh did in [this](https://github.com/AutoFixture/AutoFixture/commit/72a9cb75fa512a213f2a87f7a7455659081fb2bc) commit.